app/services/media.ts

```
@@ -370,11 +596,52 @@ export class MediaManager {
+      }
+
       return null;
     }
   }
+
+  /**
+   * Check if the browser supports getDisplayMedia for screen sharing
+   */
+  public isScreenShareSupported(): boolean {
+    // @ts-ignore: TypeScript doesn't recognize getDisplayMedia on mediaDevices
+    return !!(navigator.mediaDevices && navigator.mediaDevices.getDisplayMedia);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)